src/PlayerAuthInputPacket.php

```diff
@@ -54,10 +54,10 @@ class PlayerAuthInputPacket extends DataPacket implements ServerboundPacket{
 	private ?Vector3 $vrGazeDirection = null;
 	private int $tick;
 	private Vector3 $delta;
-	public ?ItemInteractionData $itemInteractionData;
-	private ?ItemStackRequest $itemStackRequest;
+	public ?ItemInteractionData $itemInteractionData = null;
+	private ?ItemStackRequest $itemStackRequest = null;
 	/** @var PlayerBlockAction[]|null */
-	private ?array $blockActions;
+	private ?array $blockActions = null;
```

Comment:
what does this have to do with it?

Reply:
Nothing, it's just another issue I encountered, when var dumping packet->getInteractionData I would get "Typed property cannot be accessed before initialization"

Reply:
Alvin mentioned checking flags, should I revert this?